TCMSApp/src/client/app/tests/tests.html

```diff
@@ -1,88 +1,114 @@
-                        <div class="input-group">
-                            <input type="text" class="form-control" placeholder="Search for...">
-                            <span class="input-group-btn">
-                                  <button class="btn btn-info" type="button"><span class="glyphicon glyphicon-search"></span></button>
-                            </span>
-                        </div>
-                    </div>
-                </form>
-                <h1>
-                    Tests
-                    <button type="button" class="btn btn-success btn-xs">
-                        Add Test
-                    </button>
-                </h1>
-            </div>
-            <hr>
-            <div class="row">
+<div class="matter">
+  <div class="container">
+
+<!-- header -->
+    <div class="clearfix">
+      <h1>
+        Tests
+        <form class="form-inline pull-right">
+          <input type="text" class="form-control" placeholder="Filter">
+        </form>
+      </h1>
+    </div>
+    <hr>
 
-                <!--main content-->
-                <div class="col-md-3">
-                    <ul class="list-group text-center">
-                        <li class="list-group-item">
-                            <button type="button" class="btn btn-default">
-                               Create suite
-                            </button>
-                        </li>
-                        <li class="list-group-item">Unsuited tests
-                            <button type="button" class="btn btn-default">
-                                <span class="glyphicon glyphicon-plus"></span>
-                            </button>
-                        </li>
-                        <li class="list-group-item">Suite 1
-                            <button type="button" class="btn btn-default">
-                                <span class="glyphicon glyphicon-plus"></span>
-                            </button>
-                        </li>
-                    </ul>
-                </div>
+<!--main content-->
+    <div class="row">
 
-                <div class="col-md-9">
-                    <div class="well">
-                        <div class="row">
-                            <div class="col-md-5">
-                                <div class="checkbox">
-                                    <h3>
-                                    <label>
-                                        <input type="checkbox">Test  case title
-                                    </label>
-                                    </h3>
-                                </div>
-                            </div>
-                            <div class="col-md-7"></div>
-                        </div>
+<!-- suites -->
+      <div class="col-xs-3">
+        <div class="panel panel-info">
+          <div class="panel-heading">
+            <h2>Suites <button type="button" class="btn btn-primary btn-xs" ng-click="vm.open()">NEW</button></h2>
```

Comment:
maybe something like testsVm would be better. 
Why? So that we can access parent controller by its name from any child controller